deployment/roles/deploy/defaults/main.yml

```diff
@@ -29,3 +29,26 @@ deployment: "{{ lookup('env', 'DEPLOYMENT') }}"
 
 # Whether to periodically import image(s) from registry to image stream(s)
 auto_import_images: true
+
+# Path within the worker container where the work is done
+workdir: "/workdir"
```

Comment:
You don't need the quotation marks around all the values.
I think they are needed only in case there's the templating ("{{ ... }}") or if there's a number/bool value you actually want to be a string.